.trivyignore

```diff
@@ -1,13 +1,50 @@
+# CVE-2022-0778
+# The BN_mod_sqrt() function, which computes a modular square root,
```

Comment:
No need to put the whole CVE description here. Can just look up current bulletin text when needed. If we keep the comment to just why the entry is here, it's more useful.